examples/Recipies/initialization/modules/correctly_initialized_config.py

```diff
@@ -0,0 +1,14 @@
+"""Configuration for initial_value with proper loading of the config file via the cli in the Config module"""
+
+from application_settings import ConfigBase, config_filepath_from_cli, dataclass
+
+
+@dataclass(frozen=True)
+class CorrectlyInitializedConfig(ConfigBase):
+    """Example of a Config that does not load the config in the module"""
+
+    initial_value: float = 0.0
+
+config_filepath_from_cli(CorrectlyInitializedConfig)
+print("Now loading CorrectlyInitializedConfig")
+CorrectlyInitializedConfig.load()
```

Comment:
So is there a downside to doing this? I guess we lose the possibility to easily override paths or something?

Reply:
No, I don't think there is a downside to this. Will mention it as recommended practice in the docs.